index.html

```diff
@@ -33,27 +33,27 @@ <h3>I'am a Software Developer</h3>
           class="main-icons"
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Git-hub-icon.png"
           alt="git-hub icon"
-        >
+        />
```

Comment:
![image](https://user-images.githubusercontent.com/38088818/164549334-3f9d5dbc-03ae-4340-b41c-79f7ac468b85.png)
Line 23: should be an `h1` tag since it is the largest text on the page and according to HTML guidelines, a page should have an `h1` tag.